plugins/content/caption/base/lib/api/KalturaCaptionAsset.php

```diff
@@ -61,6 +61,24 @@ class KalturaCaptionAsset extends KalturaAsset
 	 */
 	public $status;
 	
+	/**
+	 * The parent id of the asset
+	 *
+	 *@var string
+	 *@readonly
+	 *
+	 */
+	public $parentId;

```

Comment:
@shpiNi  - I think this should not be readonly because the batch should populate it when creating the captions.